node/project-management-app/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "project-management-app",
+  "version": "1.0.0",
+  "description": "A sample project management app for Google Chat",
+  "main": "index.js",
+  "scripts": {
+    "test": "mocha -p -j 2 -t 5s \"test/**/*.test.js\""
```

Comment:
I suggest reducing the code footprint by only keeping a smaller number of tests. This can be done by removing those that are redundant, their value is more in illustrating how it can be done at all levels (services, controllers, models, views) than in their completeness.

Reply:
I removed the redundant exception tests in `app-action-handler.test.js` and left a single test for them, which reduced that test file a bit. I'm not sure if I'd classify any of the other tests as redundant, as they all test different cases, even if similar.

I'm not sure I see much benefit in removing some test cases now that they're written. I don't think they difficult comprehension of the sample, as the reader can just ignore them if they're only interested in learning how the app works. However, if someone is forking or copying this sample and modifying it for learning or as a starting point for their app, I think that already having a complete set of unit tests can be very useful.

Reply:
I agree with your logic and I have no strong opinions, it's fine to keep the current version.